File: kata-solutions/video-store-kata/video-store-java/src/main/java/NewReleaseMovie.java
package com.cleancode.martinfowler.videostore;

public class NewReleaseMovie extends Movie {
  public NewReleaseMovie(String title) {
    super(title);
  }

  @Override
  public double determineAmount(int daysRented) {
    return daysRented * 3.0;
  }

  @Override
  public int determineFrequentRenterPoints(int daysRented) {
    return (daysRented > 1) ? 2 : 1;
  }
}

File: kata-solutions/video-store-kata/video-store-python/src/new_release_movie.py
from movie import Movie

class NewReleaseMovie(Movie):
  def __init__(self, title):
    super().__init__(title)

  def calculate_amount(self, days_rented):
    return days_rented * 3

  def calculate_frequent_renter_points(self, days_rented):
    if days_rented > 1:
        return 2
    return 1
